syncany-lib/src/main/java/org/syncany/operations/down/ApplyChangesOperation.java

```diff
@@ -96,15 +99,25 @@ public OperationResult execute() throws Exception {
 		}
 
 		Set<MultiChunkId> unknownMultiChunks = determineRequiredMultiChunks(actions, winnersDatabase);
-		
-		downloader.downloadAndDecryptMultiChunks(unknownMultiChunks);
+
+		try {
+			downloader.downloadAndDecryptMultiChunks(unknownMultiChunks);
```

Comment:
I feel it's more readable if they are in one try/catch block. 
Or they could be in individual methods.